xword.py

```diff
@@ -1,21 +1,45 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
-"""Crossword Solver Program"""
+"""
+Crossword Solver Program
 
-__author__ = "???"
+Description: displays possible words to fit crossword puzzle location based on
+dictionary of words
 
-# YOUR HELPER FUNCTION GOES HERE
+Author: Aaron Jackson
+Github: TimeApollo
+"""
+
+__author__ = "TimeApollo"
+import re
+
+
+def word_checker(in_word, words):
+    """checks in_word against words list, to see if any match pattern"""
+    """returns list as well as printing all matches"""
+    regex = in_word.replace(' ', '\\w')
```

Comment:
This is good.  Mind if I use your solution as the "reference" ??

Reply:
I don't mind at all as we talked yesterday. I think its cool that you like it enough to make it a solution.